src/xcode/ENA/ENAUITests/ContactJournal/ENAUITests_07_ContactJournalUITests.swift

```diff
@@ -424,6 +425,31 @@ class ENAUITests_07_ContactJournal: XCTestCase {
 		let checkinCellEmpty = overviewCellEmpty.staticTexts[AccessibilityIdentifiers.ContactDiaryInformation.Overview.checkinRiskLevelHigh]
 		XCTAssertFalse(checkinCellEmpty.isHittable)
 	}
+
+	/// Tests: ENF Risk None, Checkin Risk None, CoronaTest Added
+	func testOverviewScenario4() throws {
+		app.setLaunchArgument(LaunchArguments.infoScreen.diaryInfoScreenShown, to: true)
+		app.setLaunchArgument(LaunchArguments.risk.riskLevel, to: "low")
+		app.setLaunchArgument(LaunchArguments.contactJournal.testsRiskLevel, to: "low")
+
+		navigateToJournalOverview()
+
+		// check if overview is visible
+		XCTAssertEqual(app.navigationBars.firstMatch.identifier, app.localized("ContactDiary_Overview_Title"))
+
+		// first cell should have the text for high risk, but none about checkin
+		let overviewCellWithEncounterRisk = app.descendants(matching: .table).firstMatch.cells.element(boundBy: 1)
+		let highRiskCell = overviewCellWithEncounterRisk.staticTexts[AccessibilityIdentifiers.ContactDiaryInformation.Overview.riskLevelLow]
+		XCTAssertTrue(highRiskCell.isHittable)
```

Comment:
Could you check if waitForExistence also works and replace it, when so? Also in the other UITest